graphql/resolvers.go

```diff
@@ -2871,13 +2871,26 @@ func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.A
 				return false, err
 			}
 			for _, projectName := range perfPlugin.Projects {
-				if projectName == pRef.Id || projectName == pRef.Identifier {
-					return true, nil
+				if projectName != pRef.Id || projectName != pRef.Identifier {
+					return false, nil
```

Comment:
This seems like it would always fail on the first project if it does not match.

Reply:
absolutely, forgot to change the || to an &&. Thanks for the catch!